planner/core/explain.go

```diff
@@ -805,6 +811,11 @@ func (p *PhysicalExchangeSender) ExplainInfo() string {
 		fmt.Fprintf(buffer, "Broadcast")
 	case tipb.ExchangeType_Hash:
 		fmt.Fprintf(buffer, "HashPartition")
+	}
+	if p.CompressionMode != kv.ExchangeCompressionModeNONE {
```

Comment:
remove this if check

Reply:
We shall describe in the official doc that compression mode won't be shown if compression mode is NONE.